LevelUp.Pos.ProposedOrders.Tests/SplitTenderTests.cs

```diff
@@ -328,5 +328,60 @@ public void SplitTenderExample_NoChangeInSpendOrTax_WhenMostOfCheckIsDiscounted(
                 $"Expected: {expectedCompletedOrderValues}" + Environment.NewLine +
                 $"Actual: {completedOrderValues}");
         }
+
+        [TestMethod]
+        public void SplitTenderExample_OneCentPaid_WhenOneCentOwed()
+        {
+            // $0.01 is owed, $0.01 of that is tax. $0.00 of that is tobacco/alcohol, and the customer wants to pay
+            // $0.01 towards the check none of which is discountable.
+            PointOfSale pointOfSale = new PointOfSale(total: 2, tax: 1);
+            int exemptionAmount = 0;
+            int spendAmount = 1;
+
+            pointOfSale.ApplyTender(1);
+
+            // Create proposed order: expected values
+            AdjustedCheckValues expectedProposedOrderValues =
+                new AdjustedCheckValues(
+                    spendAmount: 1,         // amount the customer wants to pay
+                    taxAmount: 1,           // no tax will be due for this payment; tax will be paid "at the end"
```

Comment:
I think this comment is confusing since we are passing in a tax amount of `1` cent and this is the final payment for this check.

Reply:
Ahh, good catch.  I hate comments some times.  Fixed.